pkg/bootstrap/api/helpers.go

```diff
@@ -32,3 +34,19 @@ func ValidateBootstrapGroupName(name string) error {
 	}
 	return fmt.Errorf("bootstrap group %q is invalid (must match %s)", name, BootstrapGroupPattern)
 }
+
+// ValidateUsages validate if the usage is valid
```

Comment:
Help string here could be better: "Validate Usages validates that the passed in string are valid usage strings for bootstrap tokens."